### DISCUSSION TITLE
Per ECA "default" accounts in transaction

### DISCUSSION THREAD

**Discussion by sbts** [ledgersmb/LedgerSMB#5871] [Design]
Some users can't/won't update from 1.3 due to a feature that was removed to fix problems it was causing.

The feature was called....
> "transaction proposal" on AR/AP entry, 
the accounts of the last transaction with a vendor are listed when selecting that vendor

From a brief discussion on Matrix with Erik this could be replaced with a client side "defaults store"
as for a timeline and possibly getting it into 1.5.1 Erik said
> Well, since it doesn't change old code (right?), then if we have very thoroughly tested it, yes.

David G replied
> I'd hope it could be implemented template and dojo side only

**Comment by ehuelsmann** [comment]
This has been removed during the 1.3 stable period, litterally *years* ago. Not considering it a reggression anymore (just different from SL)

**Comment by ehuelsmann** [comment]
Would be nice to add it sometime during the 1.5 cycle.

**Comment by ehuelsmann** [comment]
This depends on the code for invoices being rewritten.
